code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -34,7 +37,21 @@
 		GLOB.ark_of_the_clockwork_justiciar = src
 	START_PROCESSING(SSprocessing, src)
 
-/obj/structure/destructible/clockwork/massive/celestial_gateway/proc/final_countdown(ark_time)
+/obj/structure/destructible/clockwork/massive/celestial_gateway/take_damage(damage_amount, damage_type = BRUTE, damage_flag = 0, sound_effect = 1, attack_dir)
+	. = ..()
+	if(.)
+		flick("clockwork_gateway_damaged", glow)
+		playsound(src, 'sound/machines/clockcult/ark_damage.ogg', 75, FALSE)
+		if(last_scream < world.time)
+			audible_message("<span class='boldwarning'>An unearthly screaming sound resonates throughout Reebe!</span>")
+			for(var/V in GLOB.player_list)
+				var/mob/M = V
+				if(M.z == z || is_servant_of_ratvar(M) || isobserver(M))
```

Comment:
why do ghosts need to be subjected to this if they're off the z

Reply:
Ghosts hear everything on the hierophant network, and any other servant-specific sounds and messages, so I included this for consistency.